msu/hooks/items/item_container.nut

```diff
@@ -39,6 +39,43 @@
 		this.m.Actor.getSkills().update();
 	}
 
+	o.getStaminaModifier <- function( _slots = null )
+	{
+		if (_slots == null)
+		{
+			 _slots = clone ::Const.ItemSlot;
+			 delete _slots.None;
+			 delete _slots.COUNT;
+		}
+		else
+		{
+			if (typeof _slots == "integer") _slots = [_slots];
+		}
+
+		local ret = 0;
+
+		foreach (slot in _slots)
+		{
+			local items = this.getAllItemsAtSlot(slot);
+			foreach (item in items)
+			{
+				// Avoid exceptions for items which don't have getStaminaModifier()
+				// This handles cases such as Quivers in ammo slot which don't have
+				// this function in vanilla but do in mods like Legends
+				try
+				{
+					ret += item.getStaminaModifier();
+				}
+				catch (error)
+				{
+					::MSU.Mod.Debug.printError(error + " -- for item: " + item.getID(), "debug");
```

Comment:
I have half a mind to leave the catch(error) block empty and print nothing to the log, because I think these messages will mostly be stuff that we can't do anything about and will just clog up the debug log instead of giving any useful info. They don't represent any problem in MSU so I think this printError should be removed.